src/main/scala/chisel3/ChiselExecutionOptions.scala

```diff
@@ -13,9 +14,9 @@ import firrtl.{ExecutionOptionsManager, ComposableOptions}
   * @note this extends FirrtlExecutionOptions which extends CommonOptions providing easy access to down chain options
   */
 case class ChiselExecutionOptions(
-                                   runFirrtlCompiler: Boolean = true
-                                   // var runFirrtlAsProcess: Boolean = false
-                                 ) extends ComposableOptions
+  runFirrtlCompiler : Boolean        = true,
+  scalaSimulator    : ScalaSimulator = TreadleSimulator
```

Comment:
Ditto, once the choose-a-simulator logic is gone, this is dead code

Reply:
As the person most likely to have to debug things in this area I would prefer to keep a simple switch (which, though perhaps in-obvious, is easier for me to remember than the call hierarchy to this code section) so that I can do quick comparison between treadle and the firrtl-interpreter.  I do not wish to defend this code going forward, but it is consistent at the moment with how other options are controlled and has short term utility.